<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="primaryContainer">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="stackGroupBox">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Dungeon Crawler</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QWidget" name="newProjectContainer" native="true">
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>700</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="2" column="3">
                <widget class="QPushButton" name="browseButton">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="newProjectLabel">
                 <property name="text">
                  <string>New Project Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="newProjectEdit"/>
               </item>
               <item row="6" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="locationEdit"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="locationLabel">
                 <property name="text">
                  <string>Location</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QCheckBox" name="importResourcesCheck">
                 <property name="text">
                  <string>Import Existing Resources</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1" alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="clearButton">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QPushButton" name="createButton">
                 <property name="text">
                  <string>Create New Project</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QCheckBox" name="randomMapCheck">
                 <property name="text">
                  <string>Generate Random Maps</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="8" column="1">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Meiryo UI</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Create New Project ...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="recentProjectContainer" native="true">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="3" column="0">
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="0" alignment="Qt::AlignLeft">
                <widget class="FlatButton" name="openBrowseButton">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" alignment="Qt::AlignTop">
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>Meiryo UI</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Recent Projects</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlatButton</class>
   <extends>QPushButton</extends>
   <header>flatbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
